composer.json

```diff
@@ -46,7 +46,7 @@
         "drupal/persistent_login": "~1.0",
         "drupal/phingdrushtask": "dev-7.x-2.x#f4b020f09da3c174fce40f2bd9a9617689f57124",
         "drupal/r4032login": "dev-1.x#391f10d69b9b8c466a2fddac2a6b99f1289e7781",
-        "drupal/rdf_entity": "^1.0.0-alpha12",
+        "drupal/rdf_entity": "dev-serializer",
```

Comment:
We should include this as a patch to avoid switching branches. Or at least create a new alpha version before proceeding.

Reply:
Please see my comment in the ticket.

Reply:
I've been created the upsteream issue https://www.drupal.org/project/drupal/issues/3036459 to fix this for ever.